pkg/feature/manifest_test.go

```diff
@@ -0,0 +1,180 @@
+package feature_test
+
+import (
+	"io/fs"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/afero"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/feature"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// AferoFsAdapter adapts an afero.Fs to fs.FS.
+type AferoFsAdapter struct {
+	Afs afero.Fs
+}
+
+// Open adapts the Open method to comply with fs.FS interface.
+func (a AferoFsAdapter) Open(name string) (fs.File, error) {
+	return a.Afs.Open(name)
+}
+
+var _ = Describe("Manifest Processing", func() {
+	var (
+		mockFS AferoFsAdapter
```

Comment:
I don't think it's mock, it's in-memory file system :) sort of stub, but maybe we should just name it as what it really is?

Reply:
true :+1: [6b72743](https://github.com/opendatahub-io/opendatahub-operator/pull/867/commits/6b727430b5c728fa77c8300bbe748bdd847405be)